resources/bolt_bash_completion.sh

```diff
@@ -0,0 +1,62 @@
+#####################################################################
+###
+### Bash autocompletion for bolt
+### include this file in your .bash_profile or .bashrc to use completion
+### "source bolt_bash_completion.sh"
+<<<<<<< HEAD
```

Comment:
It looks like you've still got some git diffs here.

Reply:
i'm sorry :-) Fixed it :-)

Reply:
You might need to make sure you push to your branch - I'm not  seeing it fixed